src/js/tracks/text-track.js

```diff
@@ -159,6 +160,8 @@ class TextTrack extends Track {
     let activeCues = new TextTrackCueList(tt.activeCues_);
     let changed = false;
     let timeupdateHandler = Fn.bind(tt, function() {
+      // This could be misleading. There is a getter on this property and if
```

Comment:
Maybe better to say something like:
> Calling `this.activeCues` for the side-effects of updating itself due to it's nature as a getter function.
Do not remove or cues will stop updating.

Or something along the lines that explains better why it's being used here.